openfe/setup/atom_mapping/lomap_scorers.py

```diff
@@ -401,27 +395,35 @@ def gen_ringdict(mol):
                         ringdictB[otherB.GetIdx()]):
                     is_bad = True
 
-    return 1 - 0.1 if is_bad else 0
+    return 0.1 if is_bad else 1.0
 
 
-def default_lomap_score(mapping: LigandAtomMapping):
+def default_lomap_score(mapping: LigandAtomMapping) -> float:
     """The default score function from Lomap2
 
-    Note
-    ----
-    Like other scores, relative to the original Lomap this is (1 - score)
-    I.e. high values are "bad", low values are "good"
+    Parameters
+    ----------
+    mapping : LigandAtomMapping
+      the mapping function to score
+
+    Returns
+    -------
+    score : float
+       A rating of how good this mapping is, from 0.0 (terrible) to 1.0 (great).
+       This score is a combination of many rules combined and considers factors such as the
+       number of heavy atoms in common, if ring sizes are changed or rings are broken,
+       or if other alchemically unwise transformations are attempted.
```

Comment:
```suggestion

    This score is a combination of many rules combined and considers factors such as the
    number of heavy atoms in common, if ring sizes are changed or rings are broken,
    or if other alchemically unwise transformations are attempted.
       
    Parameters
    ----------
    mapping : LigandAtomMapping
      The atom mapping to score.

    Returns
    -------
    score : float
       A rating of how good this mapping is, from 0.0 (terrible) to 1.0 (great).
```

For consistency with how other scorers are described (and to fix a typo: mapping function -> atom mapping)